cli/sawtooth_cli/network_command/compare.py

```diff
@@ -0,0 +1,330 @@
+    return [
+        map(SimpleBlock.from_block_dict, c.list_blocks(limit=3))
+        for c in clients
+    ]
+
+
+def print_summary(graph, tails):
+    """Print out summary and per-node comparison data."""
+    # Get comparison data
+    heads = get_heads(tails)
+    heights = get_heights(tails)
+    max_height = max(heights)
+    common_height, block_ids_at_common_height = get_common_height(tails)
+    lags = get_lags(heights, max_height)
+    common_ancestor = graph.root
+    divergences = get_divergences(heights, graph.root)
+
+    # Print summary info
+    col_1 = 8
+    col_n = 8
+    format_str = '{:<' + str(col_1) + '} ' + ('{:<' + str(col_n) + '} ') * 2
+    header = format_str.format("COMMON", "HEIGHT", "BLOCKS")
+    print(header)
+    print("-" * len(header))
+    print(format_str.format(
+        "ANCESTOR", common_ancestor.num, common_ancestor.ident[:col_n]))
+    print(format_str.format(
+        "HEIGHT", common_height, str(block_ids_at_common_height)))
+    print()
+
+    # Print per-node data
+    col_1 = 6
+    col_n = 8
+    format_str = \
+        '{:<' + str(col_1) + '} ' + ('{:<' + str(col_n) + '} ') * len(tails)
+    header = format_str.format("NODE", *list(range(len(tails))))
+    print(header)
+    print('-' * len(header))
+
+    print(format_str.format("HEAD", *heads))
+    print(format_str.format("HEIGHT", *heights))
+    print(format_str.format("LAG", *lags))
+    print(format_str.format("DIVERG", *divergences))
+    print()
+
+
+def print_table(graph, tails):
+    """Print out a table of nodes and the blocks they have at each block height
+    starting with the common ancestor."""
+    node_count = len(tails)
+
+    # Get the width of the table columns
+    num_col_width = max(
+        floor(log(max(get_heights(tails)), 10)) + 1,
+        len("NUM"))
+    node_col_width = max(
+        floor(log(node_count, 10)) + 1,
+        8)
+
+    # Construct the output format string
+    format_str = ''
+    format_str += '{:<' + str(num_col_width) + '} '
+    for _ in range(node_count):
+        format_str += '{:<' + str(node_col_width) + '} '
+
+    nodes_header = ["NODE " + str(i) for i in range(node_count)]
+    header = format_str.format("NUM", *nodes_header)
+    print(header)
+    print('-' * len(header))
+
+    prev_block_num = -1
+    node_list = [''] * node_count
+    for block_num, _, siblings in graph.walk():
+        if block_num != prev_block_num:
+            # Need to skip the first one
+            if prev_block_num != -1:
+                print(format_str.format(prev_block_num, *node_list))
+
+            node_list.clear()
+            node_list.extend([''] * node_count)
+            prev_block_num = block_num
+
+        for block_id, node_ids in siblings.items():
+            for node_id in node_ids:
+                node_list[node_id] = block_id[:8]
+
+    # Print the last one
+    print(format_str.format(prev_block_num, *node_list))
+
+
+def print_tree(graph, tails):
```

Comment:
Remove reference to --tree if it is not implemented

Reply:
Also, delete the function